### PR TITLE
fix: update package names

### PR DESCRIPTION
**Pull request by mhmmtucan**
Updates packages names with new organization name

### CODE REVIEW

**Review by selimssevgi — CHANGES_REQUESTED** [review]
co.eightwelve.be is too generic I think. How about co.eightwelve.cdt.be? Lets say tomorrow we will have another backend service for another project, what package would we call that one?

**Review by berkaygiris — COMMENTED** [review]
Can we add a rule for package names to checkstyle?

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <relativePath/> <!-- lookup parent from repository -->
   </parent>
 
-  <groupId>org.eightwelve</groupId>
+  <groupId>co.eightwelve</groupId>
   <artifactId>cdt-be</artifactId>
```
Comment (selimssevgi) [inline pom.xml:15]: co.eightwelve.cdt:cdt-be

Taken from spring cloud pom:
<groupId>org.springframework.cloud</groupId>
<artifactId>spring-cloud-netflix</artifactId

**Comment by mhmmtucan** [comment]
> co.eightwelve.be is too generic I think. How about co.eightwelve.cdt.be? Lets say tomorrow we will have another backend service for another project, what package would we call that one?

I thought same after pr. @selimssevgi but is it necessary to add new directory? Wouldn't co.eightwelve.cdt-be be better?

**Comment by mhmmtucan** [comment]
> Can we add a rule for package names to checkstyle?

Could you help me with that?

**Review by selimssevgi — COMMENTED** [review]
> Can we add a rule for package names to checkstyle?

Actually that's a good idea, but currently we use google_checks directly from plugin itself. we don't use any custom config xml or additions currently. For now that might not worth the effort, but for upcoming projects we could reconsider that.

**Comment by selimssevgi** [comment]
> > co.eightwelve.be is too generic I think. How about co.eightwelve.cdt.be? Lets say tomorrow we will have another backend service for another project, what package would we call that one?
> 
> I thought same after pr. @selimssevgi but is it necessary to add new directory? Wouldn't co.eightwelve.cdt-be be better?

I dont recall any usage with "-" in the package name. I am not sure even if it is legal. I think to be consistent we could continue with ...cdt.be...

**Review by selimssevgi — APPROVED** [review]

**Pull request merged**